tests/integration/test_normalization.py

```diff
@@ -0,0 +1,89 @@
+import json
+import os
+
+
+def get_test_data(name):
```

Comment:
The `-input` and `-output` format aligns with the rest of the fixtures in that directory.